### DISCUSSION TITLE
Rancher Desktop Design - Troubleshooting

### DISCUSSION THREAD

**Discussion by rak-phillip** [rancher-sandbox/rancher-desktop#694] [Archive]
## Troubleshooting

* Add context to any action we take and how it will affect the environment

![10-Troubleshooting](https://user-images.githubusercontent.com/835961/134973420-ac8d559d-b9a5-456e-bc06-6f1fafc3e1ac.png)

> Using the space to share this initial design proposal from the UI group. I've add some quick notes for each section of Rancher Desktop that might change. If you have any questions about anything you see below, please ask and I'll provide some more detail. 
>
> Original Discussion #683
>
> Feedback is highly valued and welcome.

**Comment by mook-as** [comment]
Hopefully, the users will want to reset Kubernetes (a less destructive operation) much more than factory reset (a more destructive operation).  We should probably order the items to make factory reset less attractive.

We may also want to consider making some of the items red (etc.) for the destructive operations.

We may want to add links to Slack and GitHub here, too, as an indicator of where help may be available.

**Reply by rak-phillip** [reply]
Those are all great ideas. 

Do you think it would make sense for the links to Slack & Github to be further up in the list? Perhaps, in the descriptor example where we currently have `Short explanation about when you might need to use these facilities`.

**Reply by mook-as** [reply]
Yeah, the descriptive text area sounds like a great spot for the links.
